src/main/java/formflow/library/ScreenController.java

```diff
@@ -3,29 +3,13 @@
 import com.smartystreets.api.exceptions.SmartyException;
 import formflow.library.address_validation.AddressValidationService;
 import formflow.library.address_validation.ValidatedAddress;
-import formflow.library.config.ActionManager;
-import formflow.library.config.ConditionManager;
-import formflow.library.config.FlowConfiguration;
-import formflow.library.config.NextScreen;
-import formflow.library.config.ScreenNavigationConfiguration;
-import formflow.library.config.SubflowConfiguration;
+import formflow.library.config.*;
```

Comment:
I really don't like to do this.  It makes it hard to tell which items a file is using.

Reply:
A bit old, but... https://www.javadude.com/posts/20040522-import-on-demand-is-evil/

Reply:
If we are using google style guide (our syntax is from that), then they are not use in it. https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
I have no strong opinions on it but IntelliJ does this automatically for me for some reason even though I have it turned off. Lemme try turning it off and putting it back.

Reply:
Ah, okay. You have to set the import limit to some high number - I don't think you can shut it off.